<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="imageTab">
       <attribute name="title">
        <string>Image</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="sliceIndex">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modelTab">
       <attribute name="title">
        <string>Model</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QVTKWidget2" name="qvtkwidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Surface">
    <property name="title">
     <string>&amp;Surfaces</string>
    </property>
    <addaction name="actionOpenSurface"/>
    <addaction name="separator"/>
    <addaction name="actionAnimation"/>
   </widget>
   <widget class="QMenu" name="menuImages">
    <property name="title">
     <string>Images</string>
    </property>
    <addaction name="actionOpenSource"/>
    <addaction name="actionOpenTarget"/>
    <addaction name="actionOpenSourceLabel"/>
    <addaction name="actionOpenTargetLabel"/>
   </widget>
   <widget class="QMenu" name="menuParticles">
    <property name="title">
     <string>Particles</string>
    </property>
    <widget class="QMenu" name="menuParticle_Colors">
     <property name="title">
      <string>Particle Colors</string>
     </property>
     <addaction name="actionParticleBlack"/>
     <addaction name="actionParticleRed"/>
     <addaction name="actionParticleGreen"/>
     <addaction name="actionParticleBlue"/>
     <addaction name="actionParticleWhite"/>
     <addaction name="actionParticleHSV"/>
    </widget>
    <addaction name="actionRandomParticlesInit"/>
    <addaction name="actionRunImageParticles"/>
    <addaction name="actionAnimation"/>
    <addaction name="separator"/>
    <addaction name="actionShowParticles"/>
    <addaction name="menuParticle_Colors"/>
   </widget>
   <addaction name="menuImages"/>
   <addaction name="menu_Surface"/>
   <addaction name="menuParticles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>204</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>200</width>
          <height>404</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Image Settings</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Viewing Planes</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>12</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="showXY">
              <property name="text">
               <string>XY Plane</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="showYZ">
              <property name="text">
               <string>YZ Plane</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="showZX">
              <property name="text">
               <string>ZX Plane</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Image Selection</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>12</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="showSource">
              <property name="text">
               <string>Source</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="showTarget">
              <property name="text">
               <string>Target</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showSourceLabel">
              <property name="text">
               <string>Source Label</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showTargetLabel">
              <property name="text">
               <string>Target Label</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Animation Time</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="animationTimeout">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Label Opacity</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="labelOpacity">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Zoom</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QDoubleSpinBox" name="zoomRatio">
           <property name="maximum">
            <double>4.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Particles Count</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QSpinBox" name="numberOfPoints">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>242</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>200</width>
          <height>404</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Model Settings</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>411</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="optimizerSettings">
        <attribute name="label">
         <string>Optimizer Settings</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Iterations</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="numberOfIterations">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Max Kappa</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="maxKappa">
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Sigma</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="sigma">
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Cutoff</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="cutoffDistance">
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>15.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>261</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRunImageParticles"/>
   <addaction name="actionAnimation"/>
  </widget>
  <action name="actionOpenSource">
   <property name="text">
    <string>&amp;Open Source Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionOpenTarget">
   <property name="text">
    <string>Open &amp;Target Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionOpenSurface">
   <property name="text">
    <string>Open Surface</string>
   </property>
  </action>
  <action name="actionCreateTargetSurface">
   <property name="text">
    <string>Create Target Surface</string>
   </property>
  </action>
  <action name="actionOpenSourceLabel">
   <property name="text">
    <string>Open Source Label</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="text">
    <string>Animate Trace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRunImageParticles">
   <property name="text">
    <string>Run Particles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionOpenTargetLabel">
   <property name="text">
    <string>Open Target Label</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionRandomParticlesInit">
   <property name="text">
    <string>Random Initialization</string>
   </property>
  </action>
  <action name="actionShowParticles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Particles</string>
   </property>
  </action>
  <action name="actionParticleBlack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionParticleRed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionParticleGreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionParticleBlue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionParticleWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionParticleHSV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HSV</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget2</class>
   <extends>QWidget</extends>
   <header>QVTKWidget2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
